live-examples/css-examples/pseudo-class/indeterminate.html

```diff
@@ -0,0 +1,12 @@
+<form>
+    <p>How did you find out about us?</p>
+    <label><input name="origin" type="radio" value="google"> Google</label>
+    <label><input name="origin" type="radio" value="facebook"> Facebook</label>
```

Comment:
```suggestion
    <label><input name="origin" type="radio" value="duck"> Duck-duck-go</label>
    <label><input name="origin" type="radio" value="mastodon"> Mastodon</label>
```
wanted to give a little love to the smaller players